src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import '../index.css';
+import ButtonKey from './buttonKey';
+import Display from './Display';
+
+function App() {
+  return (
+    <>
+      <Display />
```

Comment:
- [ ] You need t0 use class-based components as per the project's requirement. The objective of this milestone is to practice React class components. Kindly refactor this component to a class-based component. :pray: